ansible.sh

```diff
@@ -1,8 +1,9 @@
-#!/bin/bash -ex
+#!/bin/bash -e
 
-docker build -t ansible-boto - <Dockerfile
+docker build -t ansible-boto .
```

Comment:
Why do we need a build context now?

Reply:
oh, i didn't recognize that's what this did. should i revert it back?

Reply:
nvm, i did roll it back